### ISSUE TITLE
Windows test failure because of Cairo_jll

### ISSUE THREAD

**Issue by hyrodium** [JuliaGraphics/Luxor.jl#314]
Currently, the tests are failing on Windows when run using GitHub Actions, as illustrated in the screenshot below:

![image](https://github.com/JuliaGraphics/Luxor.jl/assets/7488140/8f4401f4-3753-4ef4-b0d4-019d0f4ab36b)

I've also observed these failures in my local Windows environment with the following setup:

- [Luxor v3.8.0](https://github.com/JuliaGraphics/Luxor.jl/tree/v3.8.0)
- Julia v1.6
- Windows

This issue does not look like it comes from the current repository code because the tests were passing when Luxor.jl v3.8.0 was released. I suspect the issue might be related to the Cairo_jll.jl package. Here's a timeline of the relevant events:

- 2024-03-08: https://github.com/JuliaGraphics/Luxor.jl/commit/0ef020715718601b9f30bb33c9166ece909cc65c, test succeeded
- 2024-03-10: [Cairo-v1.18.0+1](https://github.com/JuliaBinaryWrappers/Cairo_jll.jl/releases/tag/Cairo-v1.18.0%2B1) was released
- 2024-03-15: https://github.com/JuliaGraphics/Luxor.jl/commit/71415993d515e2ef4b2ae461d1b59793bd5facb0, test failed

**Comment by hyrodium** [comment]
Looks like https://github.com/JuliaPackaging/Yggdrasil/pull/8270 is related.

**Comment by cormullion** [comment]
Yes, I reported it [here](https://github.com/JuliaGraphics/Cairo.jl/issues/363). Since I don't use Windows, there's not much I can do...

**Comment by cormullion** [comment]
Probably a problem upstream - Cairo, Pango. Closing until more information available.
